test/Microsoft.AspNet.Mvc.WebApiCompatShimTest/ApiControllerTest.cs

```diff
@@ -419,7 +420,7 @@ public void ApiController_StatusCode()
             var result = controller.StatusCode(HttpStatusCode.ExpectationFailed);
```

Comment:
There are some APIs are on API controller that accept `System.Net.HttpStatusCode` as a parameter. We want to keep all of those and their tests the same because this is a compat shim.

Anything else we should consider changing.